index.html

```diff
@@ -59,7 +59,7 @@
         <h3>Usability</h3>
 
         <details>
-          <summary>Sandstorm is the easiest way there has ever been to run a server. Install apps as easily as on your phone.</summary>
+          <summary>Install web apps like EtherCalc or WordPress with a click. Use our hosting, or install the platform easily on your own Linux server.</summary>
```

Comment:
I think it would make a lot of sense to mention EtherCalc and WordPress in the "Easy to get apps:" point, below.

I'm wary of extending the summary text to be more than the current three lines. We spent a lot of time compacting it. I know it's really tempting to say more, but it also means fewer people will read. That's why we have the expanding sections -- so details are there for people who want them.